src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php

```diff
@@ -29,9 +29,11 @@ public function setUp(): void
     /** @test */
     public function it_gets_collection_if_api_is_enabled(): void
     {
+        $this->enableApi();
+
         static::createClient()->request(
             'GET',
-            'api/v2/admin/orders',
+            'api/v2/admin/countries',
```

Comment:
Why it needs to be changed?

Reply:
it was a quick solution for the problem, it seems like /orders endpoint was changed between 1.9 and master as it was working fine with 1.9 version.

Reply:
How does it then works in [this test](https://github.com/SirDomin/Sylius/blob/fix-disabled-api-master/src/Sylius/Bundle/ApiBundle/test/src/Tests/DisablingApiTest.php#L61)? 😱